conan/api/output.py

```diff
@@ -51,6 +51,17 @@ class ConanOutput:
     # Singleton
     _conan_output_level = LEVEL_STATUS
     _silent_warn_tags = []
+    _output_levels = {"quiet": LEVEL_QUIET,  # -vquiet 80
```

Comment:
Making this global makes more difficult to see the actual change here.

Furthermore, this dict is used only once, inside the ``define_log_level``, so it could perfectly stayed there (giving it the  minimum necessary scope, no more)

Reply:
IMO, not really. If it is used only inside one method/function of the class, it is an internal implementation detail of that method/function, no need to give it class scope.